pkg/protocols/http/httpclientpool/clientpool.go

```diff
@@ -131,6 +148,9 @@ func (c *Configuration) HasStandardOptions() bool {
 
 // GetRawHTTP returns the rawhttp request client
 func GetRawHTTP(options *protocols.ExecutorOptions) *rawhttp.Client {
+	rawHttpClientMu.Lock()
```

Comment:
What do you think about using https://pkg.go.dev/sync#OnceValue to obtain the default client once in a safe fashion?

Reply:
sync.OnceValue expects niladic func, thus making things a bit more complex. I used sync.Once instead.